tests/unit/test_service.py

```diff
@@ -248,4 +240,9 @@ def _fake_measurement_function():
 @pytest.fixture
 def measurement_service() -> MeasurementService:
     """Create a MeasurementService."""
-    return MeasurementService(_TEST_MEASUREMENT_INFO, _TEST_SERVICE_INFO)
+    assets_directory = pathlib.Path(__file__).resolve().parent.parent
```

Comment:
This isn't the assets directory. You need to add "assets" on the end.

Reply:
Moved `"assets"` from the line below to here to make the variable name make sense.